src/database.html

```diff
@@ -36,57 +36,279 @@ <h4>Size</h4>
         </ul>
     <h4>Specialty</h4>
         <ul>
-            <li>Traditional</li>
-            <li>Digital/Social Media</li>
-            <li>Interactive</li>
-            <li>Experiential</li>
             <li>Data/Technology</li>
+            <li>Digital/Social</li>
+            <li>Experiential</li>
+            <li>Interactive</li>
+            <li>Media</li>
+            <li>Traditional</li>
         </ul>
     <h4>Type of Clients</h4>
         <ul>
+            <li>Automobiles</li>
             <li>Consumer Packaged Goods</li>
-            <li>Food & Beverage</li>
-            <li>Shopper/E-Commerce</li>
             <li>Entertainment</li>
+            <li>Food & Beverage</li>
             <li>Healthcare/Pharma</li>
             <li>Insurance</li>
-            <li>Automobiles</li>
+            <li>Shopper/E-Commerce</li>
         </ul>
-    <div class="BBDO">
-        <img src="" alt=" "/>
-        <h5><strong>BBDO</strong></h5>
-        <p><strong>Locations:</strong> Atlanta, Dallas, Detroit, Los Angeles, Minneapolis, <strong>New York</strong>, San Francisco</p>
-    </div>
-    <div class="BBH">
-        <img src="" alt=" "/>
-        <h5><strong>BBH</strong></h5>
-        <p><strong>Locations:</strong> Los Angeles, New York</p>
-    </div>
-    <div class="Ogilvy & Mather">
-        <img src="" alt=" "/>
-        <h5><strong>Ogilvy & Mather</strong></h5>
-        <p><strong>Locations:</strong> Cambridge, Chicago, DC, Denver, Los Angeles, Miami, <strong>New York</strong>, Parsippany, San Francisco</p>
-    </div>
-    <div class="Goodby & Silverstein">
-        <img src="" alt=" "/>
-        <h5><strong>Goodby & Silverstein</strong></h5>
-        <p><strong>Locations:</strong> New York, <strong>San Francisco</strong></p>
-    </div>
-    <div class="Accenture Song">
-        <img src="" alt=" "/>
-        <h5><strong>Accenture Song</strong></h5>
-        <p><strong>Locations:</strong><strong> New York</strong></p>
-    </div>
-    <div class="McCann">
-        <img src="" alt=" "/>
-        <h5><strong>McCann</strong></h5>
-        <p><strong>Locations:</strong> Costa Mesa, Detroit, Miami, Minneapolis, <strong>New York</strong>, San Francisco</p>
-    </div>
-    <div class="Droga5">
-        <img src="" alt=" "/>
-        <h5><strong>Droga5</strong></h5>
-        <p><strong>Locations:</strong><strong> New York</strong></p>
-    </div>
+    <ul>
+        <li> 
+            <img src="#" alt="#"/>
+            <h5><strong>Accenture Song</strong></h5>
```

Comment:
You shouldn't need a strong tag in here since you're already using an h5 - the bold font weight comes with all heading tags by default. :) Removing it will make it easier to adjust your styles across the board later if you need to, so I'd recommend doing that before you get too much further.